examples/arquillian-jpa/pom.xml

```diff
@@ -28,10 +28,22 @@ language governing permissions and limitations under the License. -->
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.5.1</version>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
+          <source>11</source>
+          <target>11</target>
         </configuration>
       </plugin>
+      <plugin>
+       <groupId>org.apache.maven.plugins</groupId>
+       <artifactId>maven-surefire-plugin</artifactId>
+       <version>2.22.2</version>
```

Comment:
Can we use `3.0.0-M5` or higher here?

Reply:
Yes, we can, so I upgraded it to 3.0.0-M7